Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'capistrano-s3'
+gem 'dotenv-rails'
```

Comment:
Ultimately I'd like to just make a setup script that walks through installing all dependencies from node/npm through ruby/gems/npm packages.